osbs/build/build_requestv2.py

```diff
@@ -687,3 +697,38 @@ def render_name(self):
 
         # !IMPORTANT! can't be too long: https://github.com/openshift/origin/issues/733
         self.template['metadata']['name'] = name
+
+    def set_source_requests(self, reactor_config_data):
```

Comment:
IMHO property pattern would fit here better than storing those data to attributes

Reply:
I mean that there will be property `build_resources_request` in parent class always returning `None`, and  specific property implementation in `SourceBuildRequest` returning limits for source containers. It will allows us easier extension in future

Reply:
scratching property idea, to keep it consistent with limits implementation.

However, this should be renamed to `set_resource_request` (similar to `set_resource_limit`)

Should be added to parent class with empty implementation and source container specific implementation should be kept here.

Reply:
ok